.golangci.yml

```diff
@@ -7,7 +7,6 @@ run:
 linters:
   enable:
     - bodyclose
-    - deadcode
```

Comment:
Removed these [deprecated](https://github.com/caraml-dev/turing/actions/runs/7540352115/job/20524809106#step:8:47) linters. They have all since been replaced by `unused`.